docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -845,10 +852,30 @@ volume on the first run.
 
 The path of a device which is mounted for the volume.
 
+### `Driver=`
+
+Specify the volume driver name. When set to `image`, the `Image` key must also be set
+
+This is equivalent to the Podman `--driver` option
+
 ### `Group=`
 
 The host (numeric) GID, or group name to use as the group for the volume
 
+### `Image=`
+
+Specifies the image the volume is based on, when `Driver` is set the `image`.
```

Comment:
I'm not real sure what you trying to say here, here's my best guess.  Or maybe "set to an `image`"

```suggestion
Specifies the image the volume is based on when `Driver` is set to the `image`.
```